### PR TITLE
Weekly Changelog 2024-08-12 to 2024-08-19

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -140,6 +140,11 @@ Place_vending function can place unpowered machines
 Add option to make the bleeding of a pet stop
 Add different levels of vision to the overmap
 Google Maps
+Start with a wallet
+Dynamic trait purifiability
+Added mashed potatoes.
+Allow cephalopods to gain the ability to eat, sleep underwater
+Add rocks to drystone wall recipes
```
Comment (Maleclypse) [inline data/changelog.txt:147]:
```suggestion
Dynamic trait purifiability
```

File: data/changelog.txt [hunk 2]
```diff
@@ -306,6 +311,14 @@ More detailed sunset/sunrise morale based on actual weather and traits
...
 Neutral NPCs may now be shoved.
 Add game screenshots to data/screenshots
+Update some lore for the New England Community Church.
+Add chance of garbage bags in cleaning supplies for evac shelter
+v2 solar panels now spawn in the world more frequently
+Draco pistol additions
+Add more vision level (at least for FEMA camps, for now)
+Superglue Overhaul
+Added a way to fix cyborgs at rubic's castle
+Adds a .22 LR 1911
```
Comment (Maleclypse) [inline data/changelog.txt:321]:
```suggestion
Add more vision level (at least for FEMA camps, for now)
Superglue Overhaul
Added a way to fix cyborgs at rubic's castle
Added mashed potatoes.
Allow cephalopods to gain the ability to eat, sleep underwater
```

File: data/changelog.txt [hunk 3]
```diff
@@ -383,6 +396,17 @@ Zone Manager: "Show All / Hide Distant" highlights selection
...
+Fix message 'There is nothing to pickup'
+migrate uilist implementation to use ImGui
+Allow ebooks in read command
+make uilist menus render their footer text/item descriptions with color
+offset the vehicle appliance interaction menu so that it doesn't overlap with the appliance info
+Keep uilist menu items lined up with each other even if some do not have hotkeys
+wrap the text in the butchery menu
+wrap the text in the martial arts selection window
+wrap the descriptions in the EOC selector menu
+draw_colored_text should “draw” an empty line if given an empty string
```
Comment (Maleclypse) [inline data/changelog.txt:409]:
```suggestion
Removes XS/XXXL traits from the player display
Fix message 'There is nothing to pickup'
migrate uilist implementation to use ImGui
Allow ebooks in read command
```

File: data/changelog.txt [hunk 4]
```diff
@@ -597,6 +621,21 @@ Document how spellcasting proficiencies work.
...
+[Xedra Evolved] Turn activated learn-recipe Paraclesian traits into Passive ones
+Add a new psychic animal, the Direhound.
+[MoM] Move Blinding Radiance to concentration
+[Xedra Evolved] Add Purifying the Air sylph spell
+[MoM] Nether-void edits
+[MoM] Further randomize cooldowns
+[Sky Island] Add Warped Pond craft
+[Magiclysm] Update Restore Genetic Stability biomancer spell for new `MUT_INSTABILITY_MOD` effect
+XE Dreamwounds: Locations for spell item drops
+Illuminated Alchemy Perk uses
```
Comment (Maleclypse) [inline data/changelog.txt:638]:
```suggestion
[MoM] Nether Attunement affects psionic calorie usage
[Sky Island] Add statue dialogue to let you repeat construction stages
Add `CLIMB_FLYING` flag, implement it in Magiclysm, MoM and Xedra Evolved
Add a new psychic animal, the Direhound.
[Xedra Evolved] Add Purifying the Air sylph spell
[MoM] Nether-void edits
[Sky Island] Add Warped Pond craft
XE Dreamwounds: Locations for spell item drops
```

File: data/changelog.txt [hunk 5]
```diff
@@ -683,6 +722,13 @@ Add Wound Care proficiency for Park Rangers
 Opiate shakes caused by cravings go away after taking opiates. Other shakes are unaffected.
 let hospitalized humans revive as zed crawlers
 Cramped space doesn't prevent sleep
+Make aim game toolkit use less power
+rebalance beverage quench based on oral hydration index
+Greatly reduce weak point proficiency learning times.
+Cut huge atomic battery weight in half
+Adjust martial arts to have crit techs that do not require stun
+Make Frog Eyes neutral
+Exclude PARROT special attack from monster difficulty calculations
```
Comment (Maleclypse) [inline data/changelog.txt:731]:
```suggestion
rebalance beverage quench based on oral hydration index
Greatly reduce weak point proficiency learning times.
Adjust martial arts to have crit techs that do not require stun
Exclude PARROT special attack from monster difficulty calculations
```

File: data/changelog.txt [hunk 6]
```diff
@@ -898,6 +944,44 @@ Wrong header restoring saved AIM layout
...
+Recipes: Allow speech to be relearnt in the dark
+Armor repair dialogue Hub-01 fixed
+Treat hostile NPCs as unsafe even if your safe mode rules don't match.
+Treat hostile NPCs as unsafe even if your safe mode rules don't match.
+Prevent crash from -1 array idx when searching zones
+Genetic chaos mutates Perks
+Backport 71801 to the 0.H branch
+`npc::wont_hit_friend` causing segfault
+Prevent crash from using invalidated explosion data
+Out of Breath sounds not playing
```
Comment (Maleclypse) [inline data/changelog.txt:984]:
```suggestion
Reading maps will reveal previously partially revealed tiles
Prevent div by zero crash in mana widget
Mutinous NPCs no longer follower your rules
NPCs can use C4
Booby trap releases a live grenade that will actually explode
[MoM] Powers should not be directly teachable
Allow operand number formats on g++14
Prevent segfault when refitting item in spillable container
Treat hostile NPCs as unsafe even if your safe mode rules don't match.
Genetic chaos no longer mutates Perks
```

File: data/changelog.txt [hunk 7]
```diff
@@ -992,6 +1076,10 @@ Allow EOCs to check and modify battery charge in items
 clean up overmap drawing code and consolidate to one function
 Jsonized end screen
 Specify see_cost for overmap terrain with an enum instead of an integer
+Backport #71595 to the 0.H branch.
+Add Flathub download button and Flatpak manifest
+[EOC] Support user-input text in EOC (similar to num_input in `math`)
+Added support for multiple randomized field sprite variants
```
Comment (Maleclypse) [inline data/changelog.txt:1082]:
```suggestion
Add Flathub download button and Flatpak manifest
[EOC] Support user-input text in EOC (similar to num_input in `math`)
Added support for multiple randomized field sprite variants
```

**Comment by MIQted** [comment]
Can you find out why there have been no releases for Android for some time ?

**Comment by Maleclypse** [comment]
> Can you find out why there have been no releases for Android for some time ?

Make an issue and don't throw random comments on closed PRs.

**Pull request merged**